Java/Complex Flow Examples/DecryptAddReencrypt.java

```diff
@@ -0,0 +1,150 @@
+import io.github.cdimascio.dotenv.Dotenv;
+import java.io.File;
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+import okhttp3.*;
+import org.json.JSONObject;
+
+/* In this sample, we will show how to take an encrypted file and decrypt, modify
+ * and re-encrypt it to create an encryption-at-rest solution as discussed in
+ * https://pdfrest.com/solutions/create-secure-document-workflows-with-pdf-password-protection/
+ * We will be running the document through /decrypted-pdf to open the document
+ * to modification, running the decrypted result through /pdf-with-added-image,
+ * and then sending the output with the new image through /encrypted-pdf to
+ * lock it up again.
+ */
+
+public class DecryptAddReencrypt {
+
+  // Specify the path to your PDF file here, or as the first argument when running the program.
+  private static final String DEFAULT_FILE_PATH = "/path/to/file.pdf";
+
+  // Specify the path to your image file here, or as the second argument when running the
+  // program.
+  private static final String DEFAULT_IMAGE_PATH = "/path/to/file.pnf";
```

Comment:
```suggestion
  private static final String DEFAULT_IMAGE_PATH = "/path/to/file.jpg";
```
Changed this to an image file format we support.